tests/components/awair/test_sensor.py

```diff
@@ -115,7 +115,7 @@ async def test_awair_gen1_sensors(
     assert_expected_properties(
         hass,
         registry,
-        "sensor.living_room_volatile_organic_compounds",
+        "sensor.living_room_vocs",
```

Comment:
@frenck This adjusted test is the one that is now failing the CI tests."

Logging shows:
````
2023-06-27 10:32:20.794 INFO     MainThread homeassistant.helpers.entity_registry:entity_registry.py:688 Registered new sensor.awair entity: sensor.living_room_score
2023-06-27 10:32:20.795 INFO     MainThread homeassistant.helpers.entity_registry:entity_registry.py:688 Registered new sensor.awair entity: sensor.living_room_humidity
2023-06-27 10:32:20.796 INFO     MainThread homeassistant.helpers.entity_registry:entity_registry.py:688 Registered new sensor.awair entity: sensor.living_room_vocs_ratio
2023-06-27 10:32:20.797 INFO     MainThread homeassistant.helpers.entity_registry:entity_registry.py:688 Registered new sensor.awair entity: sensor.living_room_temperature
2023-06-27 10:32:20.798 INFO     MainThread homeassistant.helpers.entity_registry:entity_registry.py:688 Registered new sensor.awair entity: sensor.living_room_carbon_dioxide
2023-06-27 10:32:20.798 INFO     MainThread homeassistant.helpers.entity_registry:entity_registry.py:688 Registered new sensor.awair entity: sensor.living_room_pm2_5
2023-06-27 10:32:20.799 INFO     MainThread homeassistant.helpers.entity_registry:entity_registry.py:688 Registered new sensor.awair entity: sensor.living_room_pm10
``

So I think the sensor entity id should be:

`sensor.living_room_vocs_ratio`